### PR TITLE
Added "edwin" to the word list

### PR DESCRIPTION
**Pull request by edwincoronado**

### CODE REVIEW

File: wordle/src/main/resources/wordle-answers-alphabetical.txt [hunk 1]
```diff
@@ -632,6 +632,7 @@ ebony
 eclat
 edict
 edify
+edwin
```
Comment (edwincoronado) [inline wordle/src/main/resources/wordle-answers-alphabetical.txt:635]: this should not be here

File: wordle/src/main/resources/wordle-answers-alphabetical.txt [hunk 2]
```diff
@@ -2312,4 +2313,4 @@ young
 youth
 zebra
 zesty
-zonal
\ No newline at end of file
+zonal
```
Comment (edwincoronado) [inline wordle/src/main/resources/wordle-answers-alphabetical.txt:2316]: why is this change here? could your editor have removed an extra line?

**Review by alyssagerhart — APPROVED** [review]
